### ISSUE TITLE
icons via `autocomplete` are misaligned

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#26063]
## Description 

Sometimes the icons for websites/search suggestions appearing via `autocomplete` appear misaligned.

## Steps to reproduce 

   1. launch brave 
   2.  go into `Hamburger menu` -> `Brave Shields & privacy`
   3. enable `Auto-complete searches and URLs` & `Show suggested sites in autocomplete`
   4. start searching/visiting several websites to build a history

Eventually you'll notice that the icons are appearing misaligned as per the screenshot/example below.

## Actual result 

![alignmentIssue](https://user-images.githubusercontent.com/2602313/196101621-c8f0327e-9ccf-489e-a87d-a69ea06d07e2.png)

## Issue reproduces how often 

It's pretty easy to reproduce but not 100% sure how often. I usually see it every time I start verification on `Android`.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? `Yes`
- Can you reproduce this issue with the current Play Store Beta version? `No`
- Can you reproduce this issue with the current Play Store Nightly version? `No`

## Device details
   - Install type (ARM, x86): `ARM`
   - Device type (Phone, Tablet, Phablet): `Samsung S10+`
   - Android version: `Android 12`
   
## Brave version

```
Brave | 1.44.114 Chromium: 106.0.5249.126 (Official Build) (32-bit)
--- | ---
Revision | fb2e06d2dee20fe0c804528502b5ad72c613abd9-refs/branch-heads/5249_118@{#3}
OS | Android 12; Build/SP1A.210812.016
```

### Website problems only

- Does the issue resolve itself when disabling Brave Shields? `N/A`
- Does the issue resolve itself when disabling Brave Rewards? `N/A`
- Is the issue reproducible on the latest version of Chrome? `N/A`

**Comment by kjozwiak** [comment]
Looks like the above is fixed via `1.45.x` as I can't reproduce the issue on `1.46.x` & `1.45.x` as per the following:

**`1.44.x`** | **`1.45.x`** | **`1.46.x`**
------------|--------------|--------------
![Screenshot_20221017-023544_Brave](https://user-images.githubusercontent.com/2602313/196105753-e1bccf82-1554-45ba-a59a-344f1a45e60e.jpg) | ![Screenshot_20221017-023014_Brave - Beta](https://user-images.githubusercontent.com/2602313/196105648-dc7db0bd-24d9-44d7-8d3e-aafe5845f9c1.jpg) | ![Screenshot_20221017-022604_Brave - Nightly](https://user-images.githubusercontent.com/2602313/196105635-4fdde3a8-100f-4203-b214-d6a6114929ae.jpg)

**Comment by deeppandya** [comment]
it's fixed in 1.45.x.

**Comment by Uni-verse** [comment]
Verified on `Samsung Galaxy S21` & `Samsung Galaxy Tab S7` using the following build(s):

```
Brave	1.45.109 Chromium: 107.0.5304.62 (Official Build) (64-bit) 
Revision	1eec40d3a5764881c92085aaee66d25075c159aa-refs/branch-heads/5304@{#942}
OS	Android 12; Build/SP1A.210812.016
```
Use Case per https://github.com/brave/brave-browser/issues/26063#issue-1410973176 

1. launch brave
2. go into Hamburger menu -> Brave Shields & privacy
3. enable `Auto-complete searches and URLs` & `Show suggested sites in autocomplete`
4. start searching/visiting several websites to build a history

- Verified icons are aligned correctly in the search suggestions view
- Verified icons are aligned in both portrait and landscape view

Example | Example | Example
-|-|-
![screenshot-1666379500752](https://user-images.githubusercontent.com/17885425/197272335-e8da8842-4f59-4837-b396-c188db57c2cb.png) | ![screenshot-1666379816171](https://user-images.githubusercontent.com/17885425/197272745-2e42343a-a3ce-42a7-8c50-906aba5f731e.png) | ![screenshot-1666379513237](https://user-images.githubusercontent.com/17885425/197272338-4ac20995-ad94-4fd4-8d08-723e34ceac2f.png)

![screenshot-1666379895861](https://user-images.githubusercontent.com/17885425/197272895-8441f0d1-0c4b-4d21-bc90-ad3e1746692d.png)
